File: 01 Introdução/1041 Coordenadas de um ponto/1041.java
import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner sc = new Scanner(System.in);
    double x = sc.nextDouble();
    double y = sc.nextDouble();

    if (x != 0 && y == 0) {
      System.out.println("Eixo X");
    } else if (x == 0 && y != 0) {
      System.out.println("Eixo Y");
    } else if (x > 0 && y > 0) {
      System.out.println("Q1");
    } else if (x < 0 && y > 0) {
      System.out.println("Q2");
    } else if (x < 0 && y < 0) {
      System.out.println("Q3");
    } else if (x > 0 && y < 0) {
      System.out.println("Q4");
    } else {
      System.out.println("Origem");
    }

    sc.close();
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/1041
// Ranking 09230º, Time 0.154, Submiss Date 12/02/2023 23:33:58 (Java 14)

File: 01 Introdução/1041 Coordenadas de um ponto/1041.py
values = input().split(" ")
x, y = [float(value) for value in values]

if x != 0 and y == 0:
    print("Eixo X")
elif x == 0 and y != 0:
    print("Eixo Y")
elif y >= x > 0:
    print("Q1")
elif y > 0 > x:
    print("Q2")
elif 0 > x >= y:
    print("Q3")
elif x > 0 > y:
    print("Q4")
else:
    print("Origem")

# https://www.beecrowd.com.br/judge/pt/problems/view/1041
# Ranking 17070º, Time 0.275, Submiss Date 15/02/2023 17:26:17 (Python 3.9)
